gemfiles/rails_5_1_mongoid_7.gemfile

```diff
@@ -34,7 +34,7 @@ end
 
 group :test do
   gem "rails-controller-testing"
-  gem "simplecov", require: false
+  gem "simplecov", "< 0.18", require: false
```

Comment:
I don't know why this change but it's ok

Reply:
Newer simplecov is incompatible with codeclimate. https://github.com/codeclimate/test-reporter/issues/413